pandas/core/indexes/base.py

```diff
@@ -1644,21 +1647,235 @@ def is_unique(self) -> bool:
 
     @property
     def has_duplicates(self) -> bool:
+        """
+        Check if the Index has duplicate values.
+
+        Returns
+        -------
+        bool
+            Whether or not the Index has duplicate values.
```

Comment:
For a case like this, this might be a bit duplicative with the first line.

Do we (or the validation script) always require an explanation of the return type?

Reply:
Yes the validation script requires a description for return values. Related error code: `'RT03': 'Return value has no description'`. I confirmed this by removing one of the explanations, leaving only the return type, and the error appears when I ran `python3 scripts/validate_docstrings.py --errors=RT03`

Reply:
(this discussion is certainly not a blocker for this PR, to be clear)

@datapythonista what's your view on this? It's of course easiest to be consistent / have a clear rule in the validation. But personally, I find that it doesn't add any value in this specific case.

Reply:
I agree it's probably a bit repetitive. I think it may add value, even if from the short summary and the the name of the function, it should be easier for most people to infer what is the output (what True and False mean), I guess beginners can appreciate having it explicit. It's difficult sometimes to know if what is obvious for us it's for other people.

In any case, assuming it literally doesn't add any value, with all the work we've got with docstrings, I would just simply move forward, since there are so many other things that I think are more important and worth more our time. I think this looks fine to me, even if the repetition is not ideal.

Reply:
Duplicate content can also add noise, as you might need to read both to ensure you don't miss something. 

Anyway, not a discussion to continue on this PR